src/main/java/org/openstreetmap/atlas/utilities/scalars/Height.java

```diff
@@ -0,0 +1,45 @@
+package org.openstreetmap.atlas.utilities.scalars;
+
+import java.io.Serializable;
+
+/**
+ * Distance wrapper to denote height above or below ground
+ *
+ * @author ajayaswal
+ */
+
+public class Height implements Serializable
```

Comment:
Thinking about this class a bit more, and after a brief discussion with @MikeGost , could we potentially use or extend the ```Altitude``` class?

Reply:
Thanks, Altitude class works well here, removing Height.